db/migrate/20160720151217_create_arbitration_rules.rb

```diff
@@ -0,0 +1,16 @@
+class CreateArbitrationRules < ActiveRecord::Migration[5.0]
+  def change
+    create_table :arbitration_rules do |t|
+      t.string :type
+      t.string :name
+      t.string :parameter
+      t.string :condition
+      t.string :value
+      t.string :action
```

Comment:
I'm concerned about the generic naming on the 4 items here.  Feels like it will get confused with other things, but I'll leave that up to you.  Do you have definitions for these?